apps/api/src/test-data/commands/build-media-item-test-command-fsas.ts

```diff
@@ -14,7 +14,6 @@ const createMediaItem: CommandFSA<CreateMediaItem> = {
         aggregateCompositeIdentifier: { id, type },
```

Comment:
This is a good opportunity to leverage our new `@CoscradDataExample` decorator.

Reply:
Ok cool, I wasn't sure if this was a go globally yet.  I'll look for your example and give it a try.

Reply:
Actually, not sure how to implement this.